content/compute/boot-from-volume.md

```diff
@@ -0,0 +1,199 @@
+/*
+Title: Booting an Instance from a Volume
+*/
```

Comment:
Possibly. Looks like this now:

![image](https://cloud.githubusercontent.com/assets/38739/11094697/84e9af1a-8867-11e5-8fc2-cd16a47d1f17.png)